### PR TITLE
MGMT-2075 swagger: API connectivity check request and step

### PR DESCRIPTION
**Pull request by danielerez**
Added connectivity_check_api_request object and connectivity-check-api.
Used by the apivip_check command introduced to agent: https://github.com/openshift/assisted-installer-agent/pull/38

### CODE REVIEW

**Comment by danielerez** [comment]
@ori-amizur @yevgeny-shnaidman

File: swagger.yaml [hunk 1]
```diff
@@ -2595,6 +2596,23 @@ definitions:
       type: string
       pattern: '^([0-9]{1,3}\.){3}[0-9]{1,3}\/[0-9]|[1-2][0-9]|3[0-2]?$'
 
+  connectivity_check_api_request:
```
Comment (yevgeny-shnaidman) [inline swagger.yaml:2599]: api_vip_connectivity_check?  seems a little bit clearer

File: swagger.yaml [hunk 2]
```diff
@@ -2595,6 +2596,23 @@ definitions:
...
+    type: object
+    required:
+      - scheme
+      - host
+      - port
+    properties:
+      scheme:
+        type: string
+        description: API scheme (http/https).
```
Comment (avishayt) [inline swagger.yaml:2608]: enum?

File: swagger.yaml [hunk 3]
```diff
@@ -2595,6 +2596,23 @@ definitions:
       type: string
       pattern: '^([0-9]{1,3}\.){3}[0-9]{1,3}\/[0-9]|[1-2][0-9]|3[0-2]?$'
 
+  connectivity_check_api_request:
+    type: object
+    required:
+      - scheme
+      - host
+      - port
+    properties:
```
Comment (avishayt) [inline swagger.yaml:2605]: Do you really want it broken into scheme, host, and port, or would it make more sense to have a single string with all 3?  (Just asking, I don't know which is better for your case)

Comment (ori-amizur) [reply swagger.yaml:2605]: Agree.  A single URL string

File: swagger.yaml [hunk 4]
```diff
@@ -1855,6 +1855,7 @@ definitions:
     type: string
     enum:
       - connectivity-check
+      - connectivity-check-api
```
[the remark this answers is not in the document]
Comment (danielerez) [reply swagger.yaml]: changed.

File: swagger.yaml (hunk 1 above)
Comment (danielerez) [reply swagger.yaml:2599]: sure, changed.

File: swagger.yaml (hunk 3 above)
Comment (danielerez) [reply swagger.yaml:2603]: ok, changed to a single URL

File: swagger.yaml (hunk 2 above)
Comment (danielerez) [reply swagger.yaml]: removed, changed to a URL

**Review by filanov — APPROVED** [review]

**Pull request merged**